satpy/tests/reader_tests/test_utils.py

```diff
@@ -514,6 +514,64 @@ def test_generic_open_binary(tmp_path, data, filename, mode):
     assert read_binary_data == dummy_data
 
 
+class TestDistributed:
+    """Distributed-related tests.
+
+    Distributed-related tests are grouped so that they can share a class-scoped
+    fixture setting up the distributed client, as this setup is relatively
+    slow.
+    """
+
+    @pytest.fixture(scope="class")
+    def dask_dist_client(self):
+        """Set up and close a dask distributed client."""
+        from dask.distributed import Client
+        cl = Client()
+        yield cl
+        cl.close()
```

Comment:
It looks like dask developer's recommend using their test utilities for writing distributed-based tests:

https://distributed.dask.org/en/latest/develop.html#writing-tests

Would it be possible to use their tools instead of this?

Reply:
Looks like you might be able to import this fixture and use it:

https://github.com/dask/distributed/blob/386e5fea1cde4aefaf821e319405188266b41832/distributed/utils_test.py#L521-L525